content/content-strategy/creating-user-centred-content/content-design/top-tasks/how-to-list-top-tasks.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: text/textblock
+section: how-to-create-user-stories
+---
+## [2]Step 1: Research user tasks
+To find the top tasks for users you can use information you already have, or use existing data such as:
+- frontline staff feedback
+- surveys to staff and users
+- social media
+- interviews with key stakeholders to find out what they consider to be user top tasks
+- similar websites (to see their top tasks)
+- site behaviour analysis
+- search analysis
+
+Keep in mind that you may not have content for top tasks yet. Any research into your website analytics may show the current state, not what might be missing.
+
+## [2]Step 2: List your tasks
+Start by creating a list of all the website tasks, for example:
+- read the blog
+- lodge a tax return
+- register a business
+- contact the department
+
+Keep tasks broad, gender-neutral and free of demographics. Avoid bias.
```

Comment:
@meaghan-n are these section headings? Not sure why there are sentences spaced like this.